tests/smartTable.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { test } from '@playwright/test';
+import { SmartTablePage } from '../page-objects/smartTable';
+import { NavigationPage } from '../page-objects/navigationPage';
+
+test('Print table rows as array of objects', async ({ page }) => {
+  await page.goto('http://localhost:4200');
```

Comment:
Try to make this as the common method whenever we open the website
Likely we have a lot of test needs to open the URL before run
So could consider to use beforeEach implementation